forms/FormField.php

```diff
@@ -463,6 +463,14 @@ public function extraClass() {
 			$classes[] .= 'holder-' . $this->MessageType();
 		}
 
+		if($this->isDisabled()) {
```

Comment:
Given the generic naming here, this could throw off a lot of custom CSS/JS implementations. I'd prefer to keep it on `UploadField` and `AssetField` for now, and discuss the implications of moving this to `FormField` through a separate PR.